packages/roosterjs-content-model/test/modelApi/selection/adjustWordSelectionTest.ts

```diff
@@ -0,0 +1,280 @@
+import { adjustWordSelection } from '../../../lib/modelApi/selection/adjustWordSelection';
+import { ContentModelDocument } from '../../../lib/publicTypes/group/ContentModelDocument';
+import { ContentModelSegment } from '../../../lib/publicTypes/segment/ContentModelSegment';
+
+const defaultMarker: ContentModelSegment = {
+    segmentType: 'SelectionMarker',
+    format: {},
+    isSelected: true,
+};
+
+describe('adjustWordSelection', () => {
+    function runTest(
+        model: ContentModelDocument,
+        result: ContentModelSegment[],
+        markerIndex: number
+    ) {
+        const marker: ContentModelSegment =
+            model.blocks[0].blockType == 'Paragraph'
+                ? model.blocks[0].segments[markerIndex]
```

Comment:
So in this test we will only cover the case when selection marker is under the first block? Do you also want to test if it is under the second block, or inside a list or a table?